gitops/azure-devops/build.sh

```diff
@@ -169,6 +173,9 @@ function git_commit() {
     #Set git identity
     git config user.email "admin@azuredevops.com"
     git config user.name "Automated Account"
+    export GIT_AUTHOR_NAME="Automated Account"
```

Comment:
It is, I was trying to avoid variables that indicate what orchestrator is being used. Should I introduce them? Only then this variable can be TeamCity/AzureDevOps/Octopus/x

Reply:
Also, it won't break a pipeline in AzureDevOps even though the code introduced is just for TeamCity